db/schema.rb

```diff
@@ -81,6 +81,7 @@
     t.integer "ispf_themes", array: true
     t.string "ispf_partner_countries", array: true
     t.uuid "linked_activity_id"
+    t.integer "has_linked_activity"
```

Comment:
Has this crept in by mistake?

Reply:
I keep getting this added to my schema. Seems like we have a ghost - any ideas how to fix it?